_includes/components/charts/canvas.html

```diff
@@ -0,0 +1,6 @@
+<canvas 
+    role="img"
+    aria-labelledby="chart-heading legend-help"
```

Comment:
@brockfanning what is `chart-heading` here?

Reply:
@LucyGwilliamAdmin Good catch - I had copied that from the DAC example but then forgot to add the id. I've just pushed that commit. Hopefully this won't cause too much repetition in what the screenreader reads.

Reply:
It is a little strange because its read out again directly after but not sure if we can really do anything about it?

Reply:
So the one read out is coming from chart-heading, what is the other one coming from?

Reply:
I believe it's the `<figcaption>` that DAC recommended. Do you think we should remove that or wait and see what DAC says about it?

Reply:
I see - I've just found [this](https://www.scottohara.me/blog/2019/01/21/how-do-you-figure.html#:~:text=A%20figcaption%20provides%20a%20caption,used%20on%20the%20figure%20instead.) which may give us grounds for removing the figcaption

Reply:
I was also just starting to think maybe we could combine the h4 with the `figcaption` e.g.
![image](https://user-images.githubusercontent.com/43404358/93639322-b73e4480-f9f0-11ea-9fa3-923be014ead2.png)

but I think I remember them saying this will make the hierarchy incorrect

Reply:
Ok. Personally I would be fine with removing the figure/figcaption.

Reply:
Although the heading of the table is a caption... I think maybe one to discuss with DAC but what are your thoughts?

One thing to bare in mind with this, the captured area for the image download will need to be changed to include figcaption

Reply:
Sorry didn't refresh so didn't see your comment before I posted. I'm happy either way

Reply:
It seems so much better for without the figcaption, I removed it. It will be easy to add back if necessary.